lab-anthony/gulpfile.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const gulp = require('gulp');
+const mocha = require('gulp-mocha');
+const eslint = require('gulp-eslint');
+
+gulp.task('default', function(){
```

Comment:
Great job! In the future you may want to break eslint and mocha into separate tasks which can be combined into other tasks (like default).